### DISCUSSION TITLE
TC CLI always returns "Folder not found"

### DISCUSSION THREAD

**Discussion by iAmcDiddy** [thunderclient/thunder-client-support#1559] [Q&A]
I have updated to the latest version and am trying to run all requests in a folder.
When I run the --debug command it says 'saveToWorkspace' is undefined (although it's checked in my settings) and also 'customLocation' is undefined.
My project has a 'thunder-tests' folder and all collections defined under that.  There is a collection named "Execute_SQL_Query" and a folder in that collection named "Unauthorized".  I used the 'tab' to complete the folder names while building the command.
Below is my command prompt, command and result.  Any help is greatly appreciated.
![image](https://github.com/thunderclient/thunder-client-support/assets/6363477/95994861-3a11-4350-8ae8-62ced8c54e3e)

I have also tried not specifying the thunder-tests folder but got the same error:
![image](https://github.com/thunderclient/thunder-client-support/assets/6363477/59415998-c3f8-4674-bab4-3fa53e1d5e2c)

**Comment by rangav** [comment]
Run Folder Docs
https://docs.thunderclient.com/cli/run-collection#run-folder
